babel.config.js

```diff
@@ -0,0 +1,7 @@
+// babel.config.js
+module.exports = {
+  presets: [
+    ['@babel/preset-env', {targets: {node: 'current'}}],
```

Comment:
I think it's better to explicitly target a given version of node so it's not based on the developer's environment, as it might cause breaking changes.

Reply:
Good point